scalyr_agent/platform_posix.py

```diff
@@ -599,14 +600,50 @@ def start_agent_service(self, agent_run_method, quiet, fork=True):
         """
         self._log_init_debug("Starting agent %s" % scalyr_util.get_pid_tid())
 
+        logger = scalyr_logging.getLogger("scalyr_agent")
```

Comment:
NOTE: I know other code uses ``_log_init()`` function to make sure log lines are correctly handled before initialize phase is finished, but I don't think there is a need to use that in the signal handlers.

Here we can use logger directly, because by the time signal handlers are set up and registered, initialization phase should already be finished.

Reply:
I think that is fair.  Maybe just add a comment here to call that out?

Reply:
Agreed, added in dda3ed8.

Reply:
Thanks.